coops-ui/src/components/common/OfficeAddresses.vue

```diff
@@ -286,7 +286,7 @@ export default class OfficeAddresses extends Mixins(CommonMixin, EntityFilterMix
   private inheritRecDeliveryAddress: boolean = true
 
   // State of the checkbox for determining whether the Record address is the same as the Registered address
-  private inheritRegisteredAddress: boolean = true
+  private inheritRegisteredAddress: boolean = this.isSame(this.registeredAddress, this.recordsAddress)
```

Comment:
This probably doesn't work as you expect because this assignment is done when the component is created (I *think*) and therefore this.registeredAddress and this.recordsAddress aren't set yet.

I might be wrong but I believe I saw this situation in the BaseAddress component. If I'm right then this code looks like it does something but actually it doesn't, so it should be changed.

If you really care about the value of inheritRegisteredAddress then you may need other code (eg, watcher) to set it (unless that already exists -- I didn't check).

Is there a unit test that checks for the situation where the address are the same, and different? If not, please add.

Reply:
Also, in those unit tests, set the prop after you create the component (don't pass the props in the creation statement) to simulate what happens irl.

Reply:
I'll have to dig into it a little further. After modifying this it was working as expected but i can certainly dig a little deeper into this!  Re Unit Tests: Absolutely i can definitely update!

Reply:
This was a good topic, thanks for bringing it up!    So after digging into it, it is ok to evaluate this property with the props passed into it ( it works as expected ) BUT it was lacking being reactive ( As you mentioned ). Initially i didn't think it needed to be reactive because the props values will always be present when the component mounts however i wasn't accounting for the draft filings ( which haven't modified the base state of these addresses yet ). So inside of the addresses watcher, we re-evaluate the state of the checkbox when we're passing draft data into the offices component.       RE: Unit tests - There is tests covering when they are the same and when they are different 👍